sphinx/writers/latex.py

```diff
@@ -894,8 +886,6 @@ def visit_document(self, node):
         self.footnotestack.append(self.collect_footnotes(node))
         self.curfilestack.append(node.get('docname', ''))
         if self.first_document == 1:
-            # the first document is all the regular content ...
```

Comment:
This comment is paired with the following `first_document == 0` block. So this should not be removed.